strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/artifact/TestArtifactContext.java

```diff
@@ -209,31 +211,18 @@ private void store(Map.Entry<RepositoryPath, Path> e)
         {
             throw new UndeclaredThrowableException(ioe);
         }
-        
-        try
-        {
-            Files.delete(e.getValue());
-        }
-        catch (IOException ioe)
-        {
-            throw new UndeclaredThrowableException(ioe);
-        }
+
+        ThrowingConsumer.unchecked(Files::delete).accept(e.getValue());
     }
     
     private int repositoryPathChecksumComparator(RepositoryPath p1,
                                                  RepositoryPath p2)
     {
         Boolean isChecksumP1;
         Boolean isChecksumP2;
-        try
-        {
-            isChecksumP1 = RepositoryFiles.isChecksum(p1);
-            isChecksumP2 = RepositoryFiles.isChecksum(p2);
-        }
-        catch (IOException e)
```

Comment:
we better add `throws IOException` here with `TrwoingComparator` interface within `deployArtifact` method